### PR TITLE
Fix pd date filter issues

### PR DESCRIPTION
**Pull request by daniel-tabarcea**
- changed filter logic to not show results from the selected date
- connects unicef/etools-issues#1088

### CODE REVIEW

File: pmp/app-elements/agreements/agreement-data/agreement-list-data.html [hunk 1]
```diff
@@ -112,12 +112,12 @@
               }
 
               if (startDate && startDate.length &&
-                  (!agreement.start || agreement.start <= startDate)) {
+                  (!agreement.start || !moment.utc(agreement.start).isAfter(moment.utc(startDate)))) {
```
Comment (acory) [inline pmp/app-elements/agreements/agreement-data/agreement-list-data.html]: Shouldn't it be .isBefore?

Comment (daniel-tabarcea) [reply pmp/app-elements/agreements/agreement-data/agreement-list-data.html]: The task requested that the agreement start date must be after the filter start date for the record to be visible, and here's how the logic works: consider the Agreement starts on 01.01 and the filter start date is also set to 01.01. As per the requirements, this records should not be in the results list. 

"1 Jan is before 1 Jan"  would be false, the if condition would not trigger and the agreement would be in the result list.
"1 Jan is not after 1 Jan" would be true, the if condition would trigger and the agreement would not be in the result list.

Does this explanation make sense?

Comment (acory) [reply pmp/app-elements/agreements/agreement-data/agreement-list-data.html]: and when the agreement starts in 01.02 ?

Comment (acory) [reply pmp/app-elements/agreements/agreement-data/agreement-list-data.html]: ok, I haven't noticed the negation (!) at the begining

Comment (daniel-tabarcea) [reply pmp/app-elements/agreements/agreement-data/agreement-list-data.html]: Oh, ok

**Pull request merged**